scalafix/input/src/main/scala/migrate/NameConflict.scala

```diff
@@ -4,6 +4,6 @@ rule = [InferTypes, ExplicitImplicits]
 package migrate
 
 object NameConflict {
-  def a = null.asInstanceOf[scala.reflect.io.File]
-  def b = null.asInstanceOf[java.io.File]
+//  def a = null.asInstanceOf[scala.reflect.io.File]
+//  def b = null.asInstanceOf[java.io.File]
```

Comment:
why do you comment those tests?

Reply:
if the new rule does not apply to them then the tests can verify that no types are added

Reply:
the rulename is interesting, and I want to come with new examples. Since I do not want to rely on scala-reflect to be able to cross compile in scala 3